template/src/components/common/Map.js

```diff
@@ -38,26 +38,13 @@ const useStyles = makeStyles((theme) => ({
 
 export function Map() {
   const viewState = useSelector((state) => state.carto.viewState);
-  const [extraViewState, setExtraViewState] = useState({});
   const baseMap = useSelector((state) => baseMaps[state.carto.baseMap]);
   const dispatch = useDispatch();
   const classes = useStyles();
   let isHovering = false;
 
   const handleViewStateChange = ({ viewState }) => {
-    const {
-      longitude,
-      latitude,
-      zoom,
-      pitch,
-      bearing,
-      transitionDuration,
-      ...others
-    } = viewState;
-    setExtraViewState(others);
-    dispatch(
-      setViewState({ longitude, latitude, zoom, pitch, bearing, transitionDuration })
-    );
+    dispatch(setViewState(viewState));
```

Comment:
I did this module with @padawannn  and the previous code has a meaning. Separating the 'pure view object' from others properties allows having 2 updates:
- an update for the reducer, with pure objects
- an update in the component, in the state, allowing the use of FUNCTIONS.

That's the only way we found, as serializing functions in redux is forbidden

Reply:
After a talk with @VictorVelarde, we've agreed to keep as it's proposed in this PR because of simplificity